main.py

```diff
@@ -118,10 +118,12 @@ def evaluate(
 def selfplay(
         load_file: "The path to the network model files",
         output_dir: "Where to write the games"="data/selfplay",
+        holdout_dir: "Where to write the games"="data/holdout",
```

Comment:
is this written to local disk?

Reply:
It's written to wherever you pass it.  This can be a gs:// path